### PR TITLE
feat(providers/tts): move Rime to /ws3 WebSocket and add coda

### PR DESCRIPTION
**Pull request by seribaymadina**
1) Switch Rime from HTTP /v1/rime-tts to wss://users-ws.rime.ai/ws3 with segment=never and eos; TTFA clock starts after connect. Enable coda in matrix; add web labels and fix TTS provider display. 

2) Historical Rime TTFA on the dashboard is not comparable to pre-merge points (new protocol; handshake excluded from clock). Expect mistv3 to look much faster vs legacy HTTP measurements.

## Summary by CodeRabbit

* **New Features**
  * Rime TTS now uses the new WebSocket streaming endpoint and defaults to the Coda model/voice.

* **Bug Fixes**
  * Improved handling of interleaved protocol events so audio assembly and timing (TTFA) are more reliable.
  * Scatter chart provider assignment fixed for accurate model attribution.

* **Chores**
  * Added Coda to color and display-name configs for charts and UI.

* **Tests**
  * Updated TTS tests to exercise the WebSocket protocol and related edge cases.

[Review Change Stack](https://app.coderabbit.ai/change-stack/coval-ai/benchmarks/pull/22?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: runner/src/coval_bench/providers/tts/rime.py [hunk 1]
```diff
@@ -68,48 +67,55 @@ async def synthesize(self, text: str) -> TTSResult:
...
-        }
+        qs = urlencode(
+            {
+                "modelId": self._model,
+                "speaker": self._voice or "luna",
+                "audioFormat": "pcm",
+                "samplingRate": SAMPLE_RATE,
+                # segment=never: synthesis fires only on explicit eos, not on sentence
+                # boundary detection. Deterministic trigger; matches Deepgram Flush /
+                # Gradium end_of_stream / Hume flush patterns.
```
Comment (lorenjphillips) [inline runner/src/coval_bench/providers/tts/rime.py]: Collapse to one line — drop the cross-provider analogies, keep just `segment=never: synthesis fires only on explicit eos`

File: runner/src/coval_bench/providers/tts/rime.py [hunk 2]
```diff
@@ -68,48 +67,55 @@ async def synthesize(self, text: str) -> TTSResult:
...
+            }
+        )
+        url = f"{_WS_BASE}?{qs}"
+        headers = {"Authorization": f"Bearer {self._api_key}"}
 
         try:
-            async with aiohttp.ClientSession() as session:
+            async with ws_client.connect(url, additional_headers=headers) as ws:
+                # t0 — WS connected; /ws3 sends no server-initiated setup frame.
+                # Consistent with Hume/Cartesia/Gradium: t0 after connect, before text send.
```
Comment (lorenjphillips) [inline runner/src/coval_bench/providers/tts/rime.py]: Drop — the line above already explains t0

**Review by lorenjphillips — CHANGES_REQUESTED** [review]
A couple of comment trims — the WebSocket plumbing itself looks good.

**Review by lorenjphillips — APPROVED** [review]
